packages/openapi-to-graphql/src/types/operation.ts

```diff
@@ -173,6 +181,11 @@ export type Operation = {
    */
   isMutation: boolean
 
+  /**
+   * Whether this operation is a subscription.
+   */
+  isSubscription: boolean
+
```

Comment:
A flag to detect Subscription operation, set to true when the operation goes through `processOperationCallbacks` function.